app/Resources/translations/default/EmailsBody.xlf

```diff
@@ -1,1179 +1,2187 @@
+        <note>Line: 31</note>
       </trans-unit>
     </body>
   </file>
-  <file original="modules/ps_emailgenerator/templates/modules/ps_emailalerts/customer_qty.php" source-language="en-US" target-language="en" datatype="plaintext">
+  <file original="mails/themes/classic/modules/ps_emailalerts/customer_qty.html.twig" source-language="en-US" target-language="en" datatype="plaintext">
     <body>
-      <trans-unit id="9b096ae55ce4d2bb889392144c104069">
-        <source>{product} is now available.</source>
-        <target>{product} is now available.</target>
-        <note>Context:
-File: modules/ps_emailgenerator/templates/modules/ps_emailalerts/customer_qty.php:21</note>
-      </trans-unit>
       <trans-unit id="5b736bebe50e10240ff58d486494dc7b">
         <source>This item is once again in-stock.</source>
         <target>This item is once again in-stock.</target>
-        <note>Context:
-File: modules/ps_emailgenerator/templates/modules/ps_emailalerts/customer_qty.php:24</note>
+        <note>Line: 29</note>
       </trans-unit>
       <trans-unit id="16ce091d5fafd5b92941221eab85f3af">
         <source>You can access the product page by clicking on the link:</source>
         <target>You can access the product page by clicking on the link:</target>
-        <note>Context:
-File: modules/ps_emailgenerator/templates/modules/ps_emailalerts/customer_qty.php:25</note>
+        <note>Line: 30</note>
       </trans-unit>
       <trans-unit id="d5c8de069674d35946f2e88642635485">
         <source>You can order it right now from our online shop.</source>
         <target>You can order it right now from our online shop.</target>
-        <note>Context:
-File: modules/ps_emailgenerator/templates/modules/ps_emailalerts/customer_qty.php:26</note>
+        <note>Line: 31</note>
       </trans-unit>
     </body>
   </file>
-  <file original="modules/ps_emailgenerator/templates/modules/ps_emailalerts/new_order.php" source-language="en-US" target-language="en" datatype="plaintext">
+  <file original="mails/themes/classic/modules/ps_emailalerts/new_order.html.twig" source-language="en-US" target-language="en" datatype="plaintext">
     <body>
-      <trans-unit id="5c1cc11c191c9a0f32d16439618ed3bb">
-        <source>Customer message:</source>
-        <target>Customer message:</target>
-        <note>Context:
-File: modules/ps_emailgenerator/templates/modules/ps_emailalerts/new_order.php:312</note>
-      </trans-unit>
-      <trans-unit id="f5d74ea75357b5e139854c14f8e24fe3">
-        <source>Order details</source>
-        <target>Order details</target>
-        <note>Context:
-File: modules/ps_emailgenerator/templates/modules/ps_emailalerts/new_order.php:28</note>
+      <trans-unit id="bb09440b8cab954c036c4a45558857a5">
+        <source>Total Tax paid</source>
+        <target>Total Tax paid</target>
+     
```

Comment:
This id doesn't match the source string now.

```diff
-MD5 ("You can check for it in the [1]Advanced Parameters &gt; Logs[/1] section of your back office.") = b6fcbd2f8102d59ea3a33c67845c8236
+MD5 ("You can check for it in the [1]Advanced Parameters > Logs[/1] section of your back office.") = ad69fd3d0352d7e45ccca96a1048a747
```

Reply:
@eternoendless I reexported the catalog entirely, so now the ids are correct